src/main/scala/scorex/account/Address.scala

```diff
@@ -62,25 +56,17 @@ object Address extends ScorexLogging {
   }
 
 
-  private def isByteArrayValid(addressBytes: Array[Byte]): Boolean = {
+  private def isByteArrayValid(addressBytes: Array[Byte]): Either[InvalidAddress, Unit] = {
```

Comment:
This should probably be renamed to `validateByteArray` since it no longer returns `Boolean`